problems/secondSmallest.js

```diff
@@ -5,9 +5,22 @@
 * @param {number[]} arr - The input array of numbers
 * @returns {number} - Returns the second smallest number.
 */
+const secondSmallest = arr => {
+    if(nums.length < 2) return null
+      let firstSmallest = -1
+       let theNumberAfterFirstSmallest = -1
 
-function secondSmallest() {
-  
+    nums.forEach((num) => {
+		if(num > firstSmallest){
```

Comment:
Nice job using forEach. 
We're on the right track but need to check if num < firstSmallest because we're looking for small numbers. If it is, then we'll have to update both firstSmallest and theNumberAfterFirstSmallest